memory-db/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [dependencies]
 heapsize = { version = "0.4", optional = true }
-parity-util-mem = { version = "0.3", default-features = false }
+parity-util-mem = { version = "0.4", default-features = false }
```

Comment:
the latest version is `0.5.1`, also this is a breaking change, do you want to submit a version bump in a separate PR? https://github.com/paritytech/trie/blob/master/CONTRIBUTING.md#releasing-a-new-version

Reply:
Is it a breaking change? parity-util-mem is not a direct dependency of memory-db. In fact, you could remove the dependency and it would work. But we need it to set some flag.

Reply:
> Is it a breaking change? parity-util-mem is not a direct dependency of memory-db. In fact, you could remove the dependency and it would work. But we need it to set some flag.

Not sure what you mean, it is a direct dependency (impls `parity_util_mem::MallocSizeOf` trait)
https://github.com/paritytech/trie/blob/a43b9a19e896533bd1bf6ca1ca12a81ab9b2c455/memory-db/src/lib.rs#L424-L463

If you remove it that would be removing a trait impl, which would break downstream crates that use this feature.

Reply:
Oh right!! Sorry I badly grepped the code (I forgot to change the `-` to `_`).

I will update to version 5 and bump the version.

Reply:
Good to merge then